css/main.css

```diff
@@ -68,22 +68,26 @@ h1 {
   font-weight: bold;
   letter-spacing: -0.3px;
   color: rgb(255, 255, 255);
-  padding: 20px 51px 19px 57px;
+  padding: 20px 110px 19px 110px;

```

Comment:
I wonder why you decided to use wrong padding here and move correct padding to `.main-button_get-started`

Reply:
@probil, I made padding for button without any text and made a modifier for it with a correct paddding for `get started` text.

Reply:
@liubich but there is only one button. Why you need a special class for it?
> I made padding for button without any text 

I don't think this is useful